pyseir/inference/fit_results.py

```diff
@@ -26,6 +27,32 @@ def load_t0(fips):
     return datetime.fromtimestamp(pd.read_pickle(fit_results).set_index('fips').loc[fips]['t0_date'].timestamp())
 
 
+def convert_fips_to_str(record):
+    """
+    Returns the same record with a stringified fips key.
+    Intended to run on a single json record
+
+    Arguments:
+        record {dict} -- {'fips': {'01': 101},
+                    'R0': {'01': 3.4387109255},
+                    ... }
+
+    Returns:
+        record {dict} -- {'fips': {'01': '101'},
+                    'R0': {'01': 3.4387109255},
+                    ... }
+
+    """
+    d = {}
+    for col, vdict in record.items():
+        if col=='fips':
+            d.update({col: {k:str(v) for k,v in vdict.items()}})
```

Comment:
Any reason you're using update here?
`d[col] = {k: str(v) for k, v for vdict.items()}` seems a bit more straightforward